classes/Langchecker/DotLangParser.php

```diff
@@ -51,10 +51,13 @@ public static function getFile($path, $show_errors = true)
     public static function parseFile($path, $reference_locale = false)
     {
         $dotlang_data = [];
-        // In case the file is missing we set an empty array
+        // In case the file is missing we set an empty array for strings
         $dotlang_data['strings'] = [];
         $file_content = self::getFile($path);
 
+        // All meta tags, excluding file tags “## TAGNAME ##”
+        $meta_tags = ['## NOTE:', '## TAG:', '## MAX_LENGTH:', '## URL:'];
```

Comment:
Suggestion to avoid typing everytime the pounds and columns:
```php
$meta_tags = ['NOTE', 'TAG', 'MAX_LENGTH', 'URL'];
$meta_tags = array_map(function($v) { return "## {$v}:"; }, $meta_tags);
```